ou_dedetai/wine.py

```diff
@@ -348,39 +360,25 @@ def run_wine_proc(
 
     cmd = f"subprocess cmd: '{' '.join(command)}'"
     logging.debug(cmd)
+    if winecmd.endswith('winetricks'):
+        encoding = 'cp437'
+    else:
+        encoding = 'utf-8'
```

Comment:
Is there any case where this isn't true? The logic was more complex before, was that for a reason?

Reply:
Yes, the more complicated logic was needed to circumvent some terminal output errors in the past. However, once we switched to `run_wine_proc` sending stdout and stderr directly to wine.log, all that code starting with `with process.stdout` (old L363) was completely bypassed anyway. And no one has complained of encoding errors in the wine.log file. Either it's irrelevant now, or too obscure to have been noticeably recreated.